### ISSUE TITLE
Lock example contract's amount is too high

### ISSUE THREAD

**Issue by DavSab** [NomicFoundation/hardhat#3607]
TL;DR : Lock example contract's amount is probably higher than the eth you have in your wallet resulting in a confusing error when first trying to deploy. 

- You are starting a new project and want to setup hardhat quickly with your testnet URL and private key as environment variables.
- You try to the deploy the Lock contrat to see if everything is working
- You get an error message "INSUFFICIENT_FUNDS" from the "estimateGas" method.
- You think "I don't have enough eth to pay for gas, I should get more. 
- After 15 minutes and a quick run to [this post](https://ethereum.stackexchange.com/questions/98395/insufficient-funds-for-intrinsic-transaction-cost) you realize it is not about the gas price, it's about trying to lock in a greater amount of eth than you got in your wallet.

**Comment by fvictorio** [comment]
Thanks @DavSab, that makes sense. Do you think changing it to 0.1 ETH would be better? If so, the only changes needed are to update this line:

https://github.com/NomicFoundation/hardhat/blob/ed86a550ab606e4214fca1109ff944ada78c6bc6/packages/hardhat-core/sample-projects/javascript/scripts/deploy.js#L14

And this one:

https://github.com/NomicFoundation/hardhat/blob/ed86a550ab606e4214fca1109ff944ada78c6bc6/packages/hardhat-core/sample-projects/javascript/scripts/deploy.js#L22

And the same lines in the [typescript script example](https://github.com/NomicFoundation/hardhat/blob/ed86a550ab606e4214fca1109ff944ada78c6bc6/packages/hardhat-core/sample-projects/typescript/scripts/deploy.ts).

Feel free to send a PR with these changes if you want and have the time.

**Comment by mutedSpectre** [comment]
@fvictorio hi. Can I try to solve this PR?
